src/components/swap/AdvancedSwapDetails.tsx

```diff
@@ -63,11 +56,10 @@ export function AdvancedSwapDetails({
   const nativeCurrency = useNativeCurrency()
 
   const { expectedOutputAmount, priceImpact } = useMemo(() => {
-    if (!trade) return { expectedOutputAmount: undefined, priceImpact: undefined }
-    const expectedOutputAmount = trade.outputAmount
-    const realizedLpFeePercent = computeRealizedLPFeePercent(trade)
-    const priceImpact = getPriceImpactPercent(realizedLpFeePercent, trade)
-    return { expectedOutputAmount, priceImpact }
+    return {
+      expectedOutputAmount: trade?.outputAmount,
+      priceImpact: trade ? computeRealizedPriceImpact(trade) : undefined,
```

Comment:
do we no longer subtract LP fee percentage?

Reply:
That happens in computeRealizedPriceImpact

Reply:
this happens inside computeRealizePriceImpact. lp percentage is calculated from trade